workflows/devnet-build/Jenkinsfile

```diff
@@ -22,6 +22,12 @@ pipeline {
                     def buildEnv = dir('docker/lisk-build'){
                         return docker.build('lisk-build')
                     }
+                    // Listing contents of bucket before building because s3cmd is not available inside docker image
+                    dir ('build') {
+                        ansiColor('xterm') {
+                            sh 's3cmd ls "s3://lisk-releases/lisk-core/" > .s3_files'
```

Comment:
Why not check right here whether the remote file already exists and abort the build if so?